resources/acsl/booleanalgebra.md

```diff
@@ -32,20 +35,51 @@ It is also used to make more specific search expressions. For example, say you w
+
+And that's it! One thing to note is that when you get to longer expressions, it may be wise to break it down and use multiple columns. Let's say the expression was `X and Y or X`. This is how it could be set up:
+
+| | | A | |
+| --- | --- | --- | --- |
+| X | Y | X and Y | A or X |
+| 0 | 0 | 0 | 0 |
+| 1 | 0 | 0 | 1 |
+| 0 | 1 | 0 | 0 |
+| 1 | 1 | 1 | 1 |
+
+For this, I broke the expression into two smaller parts. I used `A` to symbolize `X and Y` simply because I didn't want to write out `X and Y or X` in the last column header.
+
+<br>
+
 # Basic Operators
 
 | Operator | How It's Denoted | Description |
 | --- | --- | --- |
 | and | `xy` or `x • y` | The resulting value is only `true` (or 1) if x and y are *both* true. |
 | or | `x + y` | The resulting value is `true` as long as *either* x or y (or both) are true. |
-| not | `x̅` or `¬x` | This is a *unary* (taking in one operand) operator. The resulting value is the opposite of x; if x is true, then the result is false, and vice versa. |
-| xor | `x ⊕ y` | The result is true if the values of x and y are different. This is also equal to `xy̅ + x̅y`; if `x = 0` and `y = 1`, then this expression would evaluate to `0 • 0 + 1 • 1`, or `1`. Since x and y have different values, this makes sense. Conversely, if `x = y = 0`, then this expression would evaluate to `0 • 1 + 1 • 0`, or `0`. |
-| xnor | `x ⊙ y` | This is the exact opposite of xor; the result is true if the values of x and y are the same. For this reason, `x ⊙ y` can also be written as `x̅ ̅⊙̅ ̅y̅`
+| not | `$\bar{x}$` or `¬x` | This is a *unary* (taking in one operand) operator. The resulting value is the opposite of x; if x is true, then the result is false, and vice versa. You can take the negations of either one or multiple terms. For multiple terms, they are implicitly grouped together by the overhead line. So, `$\bar{y + z}$` would be treated the same as `$\bar{(y + z)}$`. |
+| xor | `x ⊕ y` | The result is true if the values of x and y are different. This is also equal to `x$\bar{y}$ + $\bar{x}$y`; if `x = 0` and `y = 1`, then this expression would evaluate to `0 • 0 + 1 • 1`, or `1`. Since x and y have different values, this makes sense. Conversely, if `x = y = 0`, then this expression would evaluate to `0 • 1 + 1 • 0`, or `0`. |
+| xnor | `x ⊙ y` | This is the exact opposite of xor; the result is true if the values of x and y are the same. For this reason, `x ⊙ y` can also be written as `$\bar{x ⊕ y}$`. |
 
 For the order of precedence, it goes as follows: *not*, *and*, *xor* and *xnor*, and then *or*. 
 
-// NOTE: NEED LATEX TO DO OVERLINES SO WILL NEED TO EDIT NOT, XOR, and XNOR
```

Comment:
A note for the \bars:
If you want the bar to go over the entire expression, use \overline{expression}.